server/validators/bandNotDuplicatedForIep.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable dot-notation */
+import { registerDecorator, ValidationArguments, ValidationOptions } from 'class-validator'
+
+export default function IsNotDuplicatedForIep(validationOptions?: ValidationOptions) {
+  return (object: unknown, propertyName: string) => {
+    registerDecorator({
+      name: 'bandNotDuplicatedForIep',
+      target: object.constructor,
+      propertyName,
+      constraints: ['createJourney', 'incentiveLevels'],
+      options: validationOptions,
+      validator: {
+        validate(bandId: number, args: ValidationArguments) {
+          const existingPay = args.object['createJourney'].pay
+          return (
+            existingPay?.find(
+              // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Is the use of `eslint-disable-next-line: any` also a symptom of not having a defined type for activity pay? Not sure if it would be of benefit to many place though.